demos/browser/app/meeting/CloudWatchLogger.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Logger from '../../../../src/logger/Logger';
+import LogLevel from '../../../../src/logger/LogLevel';
+
+/**
+ * CloudWatchLogger writes logs with console
+ *
+ * ```typescript
+ *   // working with the CloudWatchLogger
+ *   const logger = new CloudWatchLogger('demo'); //default level is LogLevel.WARN
+ *   logger.info('info');
+ *   logger.debug('debug');
+ *   logger.warn('warn');
+ *   logger.error('error');
+ *
+ *   // setting logging levels
+ *   const logger = new CloudWatchLogger('demo', LogLevel.INFO)
+ *   logger.debug(debugFunc()); // this will not show up
+ *   logger.setLogLevel(LogLevel.DEBUG)
+ *   logger.debug(debugFunc()); // this will show up
+ *
+ * ```
+ */
+export default class CloudWatchLogger implements Logger {
+  name: string;
+  level: LogLevel;
+  logCapture: string[] = [];
+  meetingID: string;
+  attendeeID: string;
+  logSeqNo: any;
+
+  constructor(name: string, level = LogLevel.WARN, meetingID: string, attendeeID: string) {
+    this.name = name;
+    this.level = level;
+    this.meetingID = meetingID;
+    this.attendeeID = attendeeID;
+    this.logSeqNo = 0;
+  }
+
+  batchSizes = 50;
+  lock = false;
+  async publishToCloudWatch(base_url: string) {
+    var _this = this;
+
+    setInterval(async function () {
+
+      if (_this.lock == true)
+        return
+      _this.lock = true;
+      //if (_this.logCapture.length > 0 )
+      var batch = _this.logCapture.slice(0, _this.batchSizes);
+      const response = await fetch(
+        `${base_url}logs`, {
+          method: 'POST',
+          body: JSON.stringify({
+            logs: batch
+          })
+        }
+      );
+
+      const json = await response.json();
+      if (json.error) {
+        throw new Error(`Server error: ${json.error}`);
```

Comment:
Wouldn't the app crash if there is an error. Is that what you want? If not shouldn't we wait for next invocation.